bolt-sidecar/src/state/execution.rs

```diff
@@ -145,6 +150,7 @@ impl<C: StateFetcher> ExecutionState<C> {
     pub async fn new(
         client: C,
         max_commitments_per_slot: NonZero<usize>,
+        max_committed_gas_per_slot: NonZero<u64>,
     ) -> Result<Self, TransportError> {
```

Comment:
can we pass `Limits` as argument here instead of both the values contained in Limits?

Reply:
good point!